dll/win32/shell32/CUserNotification.cpp

```diff
@@ -76,7 +75,7 @@ VOID CUserNotification::RemoveIcon()
 {
     NOTIFYICONDATAW nid = {0};
 
-    nid.cbSize = NOTIFYICONDATAW_V3_SIZE; // sizeof(nid);
+    nid.cbSize = NOTIFYICONDATAW_V3_SIZE_COMPAT;
```

Comment:
Why not use `sizeof(nid)`? And is there any reason we couldn't use the V4 structure everywhere?

Reply:
The value of macro `_WIN32_WINNT` is changeable by target platform (XP or Vista).
`sizeof(nid)` is also changeable.
V4 is not available in XP target.